features/in_onderzoek.feature

```diff
@@ -12,109 +12,171 @@ Functionaliteit: in onderzoek
 
   Ook wanneer een gegeven geen waarde heeft en daardoor niet in het antwoord opgenomen wordt kan het in onderzoek zijn. In dat geval wordt alleen in inOnderzoek een veld opgenomen met die naam en de waarde true.
 
-  @gba
   Rule: wanneer een gegeven uit een categorie gevraagd is en er zit onderzoek op die categorie, dan wordt ook inOnderzoek geleverd
     - het gaat om groep 83 in de categorie
     - deze wordt geleverd wanneer in het antwoord ten minste één gegeven uit de categorie is gevraagd met fields
     - deze wordt ook geleverd wanneer het gevraagde gegeven geen waarde heeft en daarom niet geleverd is in het antwoord
 
-    @gba
-    Abstract Scenario: vragen om <fields> bij <in onderzoek> in onderzoek
-      Gegeven de persoon met burgerservicenummer 555550001 heeft de volgende persoonsgegevens in de registratie
-      | categorie | burgerservicenummer (01.20) | voornamen (02.10) | voorvoegsel (02.30) | geboortedatum (03.10) | aanduiding in onderzoek (83.10) | datum ingang onderzoek (83.20) | datum einde onderzoek (83.30) |
-      | 1         | 555550001                   | Arnitta           |                     | 19231213              | <waarde>                        | 20120920                       |                               |
-      En de persoon met burgerservicenummer 555550001 heeft de volgende oudergegevens in de registratie
-      | Categorie | Voornamen (02.10) | Geslachtsaanduiding (04.10) | aanduiding in onderzoek (83.10) | datum ingang onderzoek (83.20) | datum einde onderzoek (83.30) |
-      | 2         | Noa               | V                           |                                 |                                |                               |
-      | 3         | Jan               | M                           |                                 |                                |                               |
-      En de persoon met burgerservicenummer 555550001 heeft de volgende partnergegevens in de registratie
-      | Categorie | Burgerservicenummer (01.20) | Voornamen | Geslachtsaanduiding (04.10) | Datum aangaan (06.10) | Datum ontbinding (07.10) | aanduiding in onderzoek (83.10) | datum ingang onderzoek (83.20) | datum einde onderzoek (83.30) |
-      | 5         | 555550002                   | Karel     | M                           | 20091102              |                          |                                 |                                |                               |
+    Scenario: vragen om naam bij hele categorie persoon in onderzoek
+      Gegeven een persoon heeft de volgende 'persoon' gegevens
```

Comment:
is dit inderdaad de bedoeling om het zo te beschrijven: een persoon heeft persoon gegevens?

Reply:
Het komt letterlijk zo voor in de [end-to-end stap definities](https://github.com/orgs/VNG-Realisatie/teams/haal-centraal/discussions/19)

Reply:
Inmiddels heb ik deze regel op advies van @MelvLee  toch vervangen door: 

`Gegeven de persoon met burgerservicenummer '000000097' heeft de volgende gegevens`